packages/EasyApiPlatform/composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "eonx-com/easy-api-platform",
+    "description": "Provides features for API Platform applications",
+    "type": "symfony-bundle",
+    "license": "MIT",
+    "require": {
+        "php": "^8.1",
+        "api-platform/core": "^2.6 || ^2.7",
+        "symfony/dependency-injection": "^5.4 || ^6.0",
+        "symfony/http-foundation": "^5.4 || ^6.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "mockery/mockery": "^1.4"
+    },
+    "autoload": {
+        "psr-4": {
+            "EonX\\EasyApiPlatform\\": "src",
+            "EonX\\EasyApiPlatform\\Bridge\\Symfony\\": "src/Bridge/Symfony"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "EonX\\EasyApiPlatform\\Tests\\": "tests"
+        }
+    },
+    "conflict": {
+        "api-platform/core": "<2.7"
```

Comment:
We have to go this way because this package needs to be a part of the monorepo where both, 2.6 and 2.7, versions are supported. The new package will support only 2.7+ and rely on the new class structure of API Platform.